ui/app/components/ui/color-indicator/color-indicator.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import { COLORS, SIZES } from '../../../helpers/constants/design-system';
 
 export default function ColorIndicator({
-  size = 'small',
+  size = 'sm',
```

Comment:
*facepalm* can't believe I did that. can you update to SIZES.SM

Reply:
Done in 582cc1e701d5e542fc3ec105601a6b74e4c31277